data/phillip2020epidemiological/phillip2020epidemiological.yaml

```diff
@@ -0,0 +1,1061 @@
+# yaml-language-server: $schema=../.schema.yaml
+title: Epidemiological Correlates of Polymerase Chain Reaction Cycle Threshold Values
+  in the Detection of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2)
+doi: 10.1093/cid/ciaa1469
+description: "This study was conducted in Utah and Wisconsin between 23 March and\
+  \ 13 May 2020, with testing data collected during a prospective household transmission\
+  \ investigation of outpatient and mild coronavirus disease 2019 cases. \n"
+analytes:
+  N1_and_N2_probes:
+    description: >
+      SARS-CoV-2 RNA genome copy concentration calculated from evaluation of both
+      N1 and N2 probe.
+    specimen: nasopharyngeal_swab
+    biomarker: SARS-CoV-2
+    gene_target: N1 and N2 probe
+    limit_of_quantification: '3162'
```

Comment:
I think the test currently fail because this is a string rather than a number.

Reply:
Fixed with actual number. The LOD/LOQ is described as: "with a limit of detection in the range of 1000–3162 viral RNA copies per mL". While in the challenger paper, the LOD is filled with 67.32 (Calculated with ct=40). Any thought on the LOD/LOQ in yaml file?